sources/Algo/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ src/HVDCDefinitionAlgorithm.cpp
 src/LineDefinitionAlgorithm.cpp
 src/LoadDefinitionAlgorithm.cpp
 src/MainConnexComponentAlgorithm.cpp
+src/RPCLDefinitionAlgorithm.cpp
```

Comment:
I would prefer to put all the filtering process in this model and to call this DynModelFilterALgorithm